util/reggen/ip_block.py

```diff
@@ -56,6 +56,7 @@
 }
 
 OPTIONAL_FIELDS = {
+    'human_name': ['s', "human-readable name of the component"],
```

Comment:
I guess in the future it would be nice for this to be a required field as well as the `one_line_desc` as they will impact the quality of the generated docs.

Reply:
Thanks @moidx, I agree. The reason we're not making them required right away is that blocks specific to a top (`ast`, `sensor_ctrl`, `alert_handler`, and `rv_plic`) will also need entries for them, and we'll deal with those blocks in a second wave because they are less time critical.

Reply:
SGTM. Thanks for the clarification!

Reply:
Tracked in #17307 and #17308.